src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav {
+  display: flex;
+  align-items: center;
+
+  &__list {
+    display: flex;
+    text-transform: uppercase;
+    list-style: none;
+  }
+
+  &__item {
+    display: inline-block;
+    padding-top: 23px;
+    padding-bottom: 23px;
+    margin-left: 20px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    display: inline;
+    text-decoration: none;
+    color: #060b35;
+    font-weight: 500;
+    padding-top: 23px;
+    padding-bottom: 23px;
+    text-align: center;
+    position: relative;
+  }
+
+  &__link:hover {
+    color: $blue-color;
+  }
+}
+
+.is-active::after {
```

Comment:
will be better to create separate modifier for nav__link and move these styles to those modifier
and use class is-active only in html for tests passing